faraday_hal_middleware.gemspec

```diff
@@ -20,6 +20,6 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency 'bundler', '~> 1.6'
 
-  spec.add_dependency 'faraday_middleware', ['>= 0.9', '< 0.10']
+  spec.add_dependency 'faraday_middleware', ['>= 0.9', '< 1']
```

Comment:
Unless there's a specific reason I would remove the upper bound.

Reply:
updated thx